### PR TITLE
tamhom-issue49: Develop test script for below functions of Opencart web application

### PR DESCRIPTION
**Pull request by tamhom**
## Exercise
Exercise 49

## Overview
Develop test script for below functions of Opencart web application selenium

## Verification Steps
Recondition:
1. Install pycharm tool to run the test scripts.
2. Download chromedriver.exe latest version and replace the directory below with the place where you store chromedriver.exe "driver = webdriver.Chrome(executable_path='_D:\python\install\chromedriver.exe_')" in the code
3. Set up Opencart website and replace the URL below with your site:
"driver.get("_http://localhost:81/Shop/upload/_")" in the code.

Step to run:
1. Copy the codes into Pycharm.
2. Run that project or run with below commands:

python ./selenium_forget_password.py
python ./selenium_login.py
python ./selenium_register.py

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<component name="InspectionProjectProfileManager">
```
Comment (duynguyenx) [inline .idea/inspectionProfiles/Project_Default.xml]: Pls remove this file

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <profile version="1.0">
```
Comment (duynguyenx) [inline .idea/inspectionProfiles/Project_Default.xml]: Pls also remove this file

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```
Comment (duynguyenx) [inline .idea/inspectionProfiles/profiles_settings.xml]: also delete this one

**Comment by duynguyenx** [comment]
@tamhom 
1 . pls correct the pull request description.
2. We have 3 test cases for 3 functions. So we should also have 3 test scripts and can be executed 1 by 1.

File: python_basic/tamhom/selenium_login_register.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__author__ = 'tamhom'
```
Comment (duynguyenx) [inline python_basic/tamhom/selenium_login_register.py]: Pls remove this file

Comment (tamhom) [inline python_basic/tamhom/selenium_login_register.py]: Hi a @duynguyenx ,
I've removed these directories. Please help to review again and contact me if something are wrong.

**Pull request merged**